test/integration/test-amp-inabox.js

```diff
@@ -60,7 +60,7 @@ function writeSafeFrame(doc, iframe, adContent) {
  * iframes that has been removed when their tests ended.
  */
 function unregisterIframe(frame) {
-  const hostWin = window.top;
+  const hostWin = window;
```

Comment:
Do we have a test that shows that we can handle talking to a host script that isn't at window.top and isn't at the immediate parent?

Reply:
Added.